components/event-sources/reconciler/testing/factory.go

```diff
@@ -61,6 +62,7 @@ func MakeFactory(ctor Ctor) rt.Factory {
 		ctx, sourcesClient := fakeclient.With(ctx, ls.GetSourcesObjects()...)
 		ctx, servingClient := fakeservingclient.With(ctx, ls.GetServingObjects()...)
 		ctx, eventingClient := fakeeventingclient.With(ctx, ls.GetEventingObjects()...)
+		fakeAuthV1Alpha1Client := fakeclientsetauthv1alpha1.NewSimpleClientset()
```

Comment:
The generator didn't produce a helper like the ones above?

Reply:
True, these are remains of the previous approach. Refactored as per injection clients.